housekeeping-vacuum-tool/src/main/java/com/hotels/housekeeping/tool/vacuum/conf/TablesValidation.java

```diff
@@ -0,0 +1,32 @@
+package com.hotels.housekeeping.tool.vacuum.conf;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties
+public class TablesValidation {
```

Comment:
why is it that we are not having to provide: @ConfigurationProperties(prefix = "tables-validation") here ?

Reply:
Also, I feel this class will be better named as TablesValidationConfig. I understand we don't have to specify it as config coz you have annotated it as Configuration but in the code when you read the following:

TablesValidator validator = new TablesValidator(emptyTablesValidation);

Its a little confusing as to what is that emptyTablesValidation doing. Not a biggie, just want to know other people's thoughts.

Reply:
The prefix is default if I am not mistaken.
I chose name because all the other "config" classes are like that so I kept that style. But I slightly prefer TablesValidationConfig as well. Other people have strong opinions about this shall I change?

Reply:
I'll rename it